games/redeclipse/Portfile

```diff
@@ -30,6 +29,8 @@ if {$subport eq ${name}} {
     PortGroup       app 1.0
 
     github.setup    ${name} base ${version} v
+    revision        1
```

Comment:
Why does the revision need to be increased?

Reply:
The previous revision successfully built on the builders, so I was under the impression that they wouldn't run through the build and upload the binaries to packages.macports.org unless I revbumped. Is that not the case? If it's not, I can remove the revbump.

Reply:
That's not the case. All commits notify the builders, and they decide if a build is necessary. A change in distributability due to a change of license does trigger a build.

Reply:
Oh, I didn't know that. Ok, I'll remove the revbump, then.

Reply:
See the documentation for `revision` at https://guide.macports.org/#reference.keywords.